common/test/acceptance/tests/lms/test_account_settings.py

```diff
@@ -259,16 +261,16 @@ def test_full_name_field(self):
         self._test_text_field(
             u'name',
             u'Full Name',
-            self.username,
```

Comment:
This code and below was relying on the fact that the full_name was always the same as the username. That's no longer the case.